bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/BulkPageTagger.java

```diff
@@ -0,0 +1,254 @@
+/*-
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2023 AEM developer community
```

Comment:
CodeClimate still complains on this line. It should read Adobe, not "AEM developer community"
`Copyright (C) 2013 - 2023 Adobe`

please update the licence header in all the files